src/test/scala/com/linagora/openpaas/gatling/OpenPaaSMixSimulation.scala

```diff
@@ -32,12 +32,14 @@ class OpenPaaSMixSimulation extends Simulation {
     .pause(1 second)
     .exec(provisionContacts)
     .pause(1 second)
+    .exec(logout)
+    .pause(1 second)
     .during(ScenarioDuration) {
       randomSwitch(
-        25.0 -> exec(SendMessageScenari.generate()),
-        25.0 -> exec(SearchEventsScenari.generate()),
-        25.0 -> exec(SendEmailScenari.generate()),
-        25.0 -> exec(OpenContactScenari.generate())
+        //25.0 -> exec(SendMessageScenari.generate()),
+        33.3 -> exec(SearchEventsScenari.generate()),
+        33.3 -> exec(SendEmailScenari.generate()),
+        33.3 -> exec(OpenContactScenari.generate())
```

Comment:
I think I do spend more time on email than on other stuff to be honnest

Reply:
I do other stuff in the webmail than just sending emails.

Any plan to add more actions?